### ISSUE TITLE
Cannot reach this link via keyboard

### ISSUE THREAD

**Issue by alexandrudima** [Microsoft/vscode#23382]
Tab from the search goes straight to the list:

![image](https://cloud.githubusercontent.com/assets/5047891/24400315/45f7714a-13b0-11e7-8a2a-56fafd20acb9.png)
